template/src/screens/DemoScreen.tsx

```diff
@@ -29,33 +21,13 @@ interface DemoScreenProps {
 }
 
 const DemoScreen = ({ navigation }: DemoScreenProps) => {
-  const [isLogoutLoading, setIsLogoutLoading] = useState(false)
   const counter = useAppSelector(selectCounter)
-  const comicRequest = useAppSelector(selectComic)
   const dispatch = useAppDispatch()
   const { t } = useTranslation()
 
-  useEffect(() => {
-    if (isNotRequested(comicRequest)) {
-      dispatch(getLatestComicAsync())
-    }
-  }, [comicRequest.state])
+  const { isLoading, data: comicData } = useLatestComicQuery()
 
-  const logOut = async () => {
-    try {
-      setIsLogoutLoading(true)
-      // typical logout for apps that require user to be logged in
-      // before giving any further access
-      persistor.pause()
-      await clearPersistence()
-      dispatch(resetStore())
-      persistor.persist()
```

Comment:
Cleanup logic moved to `useLogOutMutation`, which is in `template/src/remote/auth.ts`.

Also, no need to set loading states, because it's handled internally by lib.